recipes/plip/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "plip" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7477d8d427c1301ad1a7b8c2c11608d27a6d42a2ac2f5197cdf4c4fe152ca05
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - future
+    - numpy
+    - openbabel
+    - lxml
+
+test:
+  imports:
+    - plip
+  # commands:
+  #   - plip --help
```

Comment:
did you want to test this?

Reply:
yes but the command is not specified in the setup.py yet. I will be in the next version. I will enable it at that time.